apps/src/sites/studio/pages/googleBlocklyWrapper.js

```diff
@@ -245,9 +245,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
   // so that our code generation logic still works with Google Blockly
   blocklyWrapper.Generator.blockSpaceToCode = function(name, opt_typeFilter) {
     const generator = blocklyWrapper.getGenerator();
-    if (!generator.isInitialized) {
-      generator.init(blocklyWrapper.mainBlockSpace);
-    }
+    generator.init(blocklyWrapper.mainBlockSpace);
```

Comment:
The Google Blockly generator clears its state each time you generate code, and expects that if you want to re-generate code you will reinitialize. This is different from our generator which maintained its state for the entire duration, and initializing was not idempotent so you only wanted to do it once at the beginning.

This didn't really matter for Flappy because there are no variables, but in Sprite Lab, the variable map in the generator was getting cleared after the first preview, and never reinitialized, so subsequent previews/runs weren't working.